tutorial/sourcemodel.md

```diff
@@ -69,18 +69,43 @@ If it worked well, you will see the coordinate system specified in the mri struc
     cfg.coordsys = 'ctf';
     mri          = ft_volumerealign(cfg, mri);
 
+#### Optional step: Include headshape information
+
+If you have collected additional information about the location of headposition markers (for example by means of the Polhemus), you can refine your alignment accordingly. For this you can use **[ft_read_headshape](https://github.com/fieldtrip/fieldtrip/blob/release/ft_read_headshape.m)** and **[ft_volumerealign](https://github.com/fieldtrip/fieldtrip/blob/release/ft_volumerealign.m)** with method 'headshape' to either automatically or interactively rotate, scale and translate the mri until it matches the recorded headshape best. In this example we use information about the headshape for illustration even though it does not add additional information beyond the mri. The output will contain an updated transformation matrix.
+
+  hs = ft_read_headshape('Subject01.shape');
```

Comment:
perhaps not refer to this particular file, but use a rather generic <filename>. Then, explain that for this subject we don't have a polhemus measurement ,but that we include the example code nonetheless.

Reply:
since the `Subject01.shape` is actually shared, and we don't have another (better) example, I think we should just use this one here. I explained that it is not from a Polhemus but from MRIViewer.